target/linux/ath79/image/generic.mk

```diff
@@ -676,6 +676,15 @@ define Device/librerouter_librerouter-v1
 endef
 TARGET_DEVICES += librerouter_librerouter-v1
 
+define Device/mikrotik_rb-wapg-5hact2hnd
+  $(Device/rb-nor-flash-16M-ac)
+  ATH_SOC := qca9556
+  DEVICE_MODEL := RouterBOARD wAP G-5HacT2HnD (wAP AC)
```

Comment:
What is the "wAP AC" referring to here? Should we introduce an alternative name?

Reply:
MikroTik calls it "wAP AC" (see [https://mikrotik.com/product/RBwAPG-5HacT2HnD](https://mikrotik.com/product/RBwAPG-5HacT2HnD)).

We could also use `DEVICE_MODEL := wAP AC (RouterBOARD wAP G-5HacT2HnD)`... I don't really have a strong opinion about it.

Reply:
Well, you could do
DEVICE_MODEL := RouterBOARD wAP G-5HacT2HnD
DEVICE_ALT0_VENDOR := Mikrotik
DEVICE_ALT0_MODEL := RouterBOARD wAP AC

in case the latter name is sufficiently unique.

make menuconfig will then show both names.

Reply:
:)